eclipse-theia/theia
# Allow specifying max lines used for AI code completion context

Review https://github.com/eclipse-theia/theia/pull/14537 first, please

#### What it does

Adds a new user setting allowing to specify max lines used for AI code completion context

#### How to test

- Change the setting to other values than 0
- Open the AI History View => Code Completion
- Trigger code completion (SHIFT+SPACE) and check the context

#### Follow-ups

This could be more sophisticated, e.g. with percentages for prefix and suffix or by respecting the AST.
#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
